src/apps/queues/models.py

```diff
@@ -0,0 +1,53 @@
+from urllib.parse import urlparse
+
+from django.conf import settings
+from django.db import models
+from pyrabbit.http import HTTPError
+# from django.contrib.sites.models import Site
+
+import uuid
+
+from apps.queues import rabbit
+
+
+class Queue(models.Model):
+    name = models.CharField(max_length=64)
+    vhost = models.UUIDField(unique=True)
+    is_public = models.BooleanField(default=False)
+    owner = models.ForeignKey(
```

Comment:
why is owner nullable? Shouldn't a Queue owner be required?

Reply:
If someone makes a queue, attaches it to a competition, then deletes their account, the idea is that this queue will stay alive and competitions using it won't be harmed